tutorials/plugins/android/android_plugin.rst

```diff
@@ -250,6 +253,22 @@ Java will most likely run in a separate thread, so calls are deferred:
 
 Godot will detect this singleton and initialize it at the proper time.
 
+Using it from GDScript
+^^^^^^^^^^^^^^^^^^^^^^
+
+First you will need to add your singleton into the android modules to be loaded. Go to "Project > Project Settings".
+Then on the tab "General" go to the "Android" section, and fill the Modules part with your module name. 
+The module should include the full Java path. For our example: `org/godotengine/godot/MySingleton`
```

Comment:
Should be two backticks on each side in reST, contrarily to Markdown. And add a dot at the end. Otherwise it looks good :)

Reply:
Ah yes, it's way better with `` :)
![image](https://user-images.githubusercontent.com/430679/65669193-65641800-e043-11e9-8eb0-f92a9ebf59c8.png)